tests/EmitTest.elm

```diff
@@ -29,30 +61,12 @@ javascript =
                             |> typed
                             |> JS.emitExpr
                             |> Expect.equal output
-
-            typed : Typed.Expr_ -> Typed.Expr
-            typed expr =
-                ( expr, Type.Int )
-
-            typedInt : Int -> Typed.Expr
-            typedInt int =
-                typed (Literal (Int int))
-
-            typedString : String -> Typed.Expr
-            typedString str =
-                typed (Literal (String str))
-
-            typedBool : Bool -> Typed.Expr
-            typedBool bool =
-                typed (Literal (Bool bool))
           in
-          describe "emitExpr"
+          describe "emitExpr_"
```

Comment:
looking at what gets run inside `runTest`, this should probably be `emitExpr` instead of `emitExpr_`